src/Components/Nav/Nav.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+import './nav.css';
+
+const Nav = () => {
+    return(
+        <nav className="nav">
+            <ul className="nav__list">
+                <li className="nav__item"><a href="" className="nav__link">List of tasks</a></li>
+                <li className="nav__item"><a href="" className="nav__link">Finished tasks</a></li>
+                <li className="nav__item"><a href="" className="nav__link">Add task</a></li>

```

Comment:
You should make a separate component for list items and pass data with the props (for example title and destination)

Reply:
You can also create an array in src/constants/navigation.ts with TITLE and TO props and then u can map this array and render whole navigation

Reply:
About styles, i added a package: bem-css-modules and when i write styles i use variable to call styles.

Example
<p className={style('title')}></p>